packages/app-project/stores/Project.spec.js

```diff
@@ -0,0 +1,73 @@
+import Project from './Project'
+import Store from './Store'
+import stubPanoptesJs from '../test/stubPanoptesJs'
+import projectFixture, { initialState } from '../test/fixtures/project'
+
+let rootStore
+let projectStore
+
+describe.only('Stores > Project', function () {
+  it('should exist', function () {
+    rootStore = Store.create()
+    projectStore = rootStore.project
+    projectStore.should.not.be.undefined
+  })
+
+  describe('default model properties', function () {
+    before(function () {
+      rootStore = Store.create()
+      projectStore = rootStore.project
+    })
+
+    it('should have a displayName property', function () {
+      projectStore.displayName.should.equal('')
+    })
+
+    it('should have an error property', function () {
+      expect(projectStore.error).to.equal(null)
+    })
+
+    it('should have an id property', function () {
+      projectStore.id.should.equal('')
+    })
+
+    it('should have a state property', function () {
+      projectStore.state.should.equal('initialised')
+    })
+
+    after(function () {
+      rootStore = null
+      projectStore = null
+    })
+  })
+
+  describe('fetch method', function () {
+    before(function () {
+      rootStore = Store.create({}, { client: stubPanoptesJs })
+      projectStore = rootStore.project
+    })
+
+    it('should exist', function () {
+      projectStore.fetch.should.be.a('function')
+    })
+
+    it('should fetch a valid project resource', function (done) {
+      projectStore.fetch('foo/bar')
```

Comment:
I was going to comment that you can use a return here so that mocha knows this is an async test, but `fetch()` isn't returning a Promise?

I suppose you could do this instead without the use of the `done()` callback:
```
return Promise.resolve(projectStore.fetch())
```

Or return `project` in the fetch method so that a Promise is being returned?

Reply:
It does return a promise, but if you [have a look at the lines below](https://github.com/zooniverse/front-end-monorepo/pull/22/files/ecbff9eee11e8399ddf9b1ed8b7ebb6515b9e81e#diff-6d1a0d34d2c8989c64ff26747cce88b8), we can test the loading state as well as the resolved result

Reply:
It does return a promise, but if you [have a look at the lines below](https://github.com/zooniverse/front-end-monorepo/pull/22/files/ecbff9eee11e8399ddf9b1ed8b7ebb6515b9e81e#diff-6d1a0d34d2c8989c64ff26747cce88b8), we can test the loading state as well as the resolved result

Reply:
Ah, yeah I was misreading it.